java/libraries/io/src/processing/io/LED.java

```diff
@@ -141,7 +141,7 @@ public void close() {
    *  Sets the brightness
    *  @param bright 0.0 (off) to 1.0 (maximum)
    */
-  public void set(float bright) {
+  public void setBrightness(float bright) {
```

Comment:
`brightness()` would be the correct name here (assuming there aren't other brightness-related methods planned).